mllib/src/test/java/org/apache/spark/mllib/clustering/JavaLDASuite.java

```diff
@@ -110,7 +111,16 @@ public void distributedLDAModel() {
 
     // Check: topic distributions
     JavaPairRDD<Long, Vector> topicDistributions = model.javaTopicDistributions();
-    assertEquals(topicDistributions.count(), corpus.count());
+    // SPARK-5562. since the topicDistribution returns the distribution of the non empty docs
+    // over topics. Compare it against nonEmptyCorpus instead of corpus
+    JavaPairRDD<Long, Vector> nonEmptyCorpus = corpus.filter(
+            new Function<Tuple2<Long, Vector>, Boolean>() {
```

Comment:
thanks @jkbradley. I think you meant scala unit test. Please note that the org test was testing for the topicDist vs corpus count. also we test other things like log likelyhood etc in both scala and java unittest.